### ISSUE TITLE
training results much better than the paper?

### ISSUE THREAD

**Issue by tanglw2020** [ArchipLab-LinfengZhang/pytorch-self-distillation-final#2]
I trained the resnet18, and got
[epoch:187, iter:73117] Loss: 9.337 | Acc: 4/4: 87.66% 3/4: 83.10% 2/4: 76.07%  1/4: 70.88% Ensemble: 85.67%
Waiting Test!
Test Set AccuracyAcc: 4/4: 80.7200% 3/4: 80.4800% 2/4: 78.5100%  1/4: 75.5400% Ensemble: 81.5000%

the results obviously much better than the paper,  this is right?

**Comment by ArchipLab-LinfengZhang** [comment]
Yes. It has much better results than our ICCV2019 paper. We have improved self-distillation in our journal paper, which be relased.

**Comment by tanglw2020** [comment]
really great job

**Comment by ArchipLab-LinfengZhang** [comment]
: )

**Comment by laisimiao** [comment]
Hi @ArchipLab-LinfengZhang
Is journal version paper available publicly?
